testdata/enum_expressions.zs

```diff
@@ -16,8 +16,8 @@ enum uint32 CrazyEnum
     DEC_EXPR = 835,
     ADD_EXPR = 2 + 6,
     SUB_EXPR = 10 - 5,
-    MULT_EXPR = 1 * 7,
-    DIV_EXPR = 10 / 4,
+    MULT_EXPR = 2 * 7,
+    DIV_EXPR = 10 / 3,
```

Comment:
Any reason for changing these?

Could we add extra unit tests, that ensure that the generated methods actually work?

Reply:
Without this change multiple enums had the same value, which results in a compile error for the value-to-name map having duplicated keys.

Reply:
This means that this PR introduces a breaking change as we cannot support enum aliases anymore. Is this OK to do?

Reply:
The original example was broken - it doesn't make sense to have a single number map to two names for a serialisation format.